build.gradle

```diff
@@ -250,6 +250,7 @@ configure(allprojects) { project ->
 		repositories {
 			mavenCentral()
 			maven { url "https://repo.spring.io/libs-spring-framework-build" }
+			maven { url "https://repo.spring.io/libs-milestone-local"}
```

Comment:
This should be `repo.spring.io/milestone` FTR.

Reply:
fair point, fixed